scalding-core/src/main/scala/com/twitter/scalding/typed/KeyedList.scala

```diff
@@ -123,9 +124,9 @@ trait KeyedListLike[K, +T, +This[K, +T] <: KeyedListLike[K, T, This]]
   /**
    * Use Algebird Aggregator to do the reduction
    */
-  def aggregate[B, C](agg: Aggregator[T, B, C]): This[K, C] =
+  def aggregate[B, C](agg: Aggregator[T, B, C])(implicit q: Quoted): This[K, C] =
```

Comment:
I don't think we can look inside the aggregrator and filter, can we? Can we really use the quoted meaningfully?